### PR TITLE
[WIP] added missing 'partition' argument to slurm launcher

### PR DESCRIPTION
**Pull request by kkmann**
# Prework

* [x] I understand and agree to the [Contributor Code of Conduct](https://github.com/wlandau/crew.cluster/blob/main/CODE_OF_CONDUCT.md).
* [x] I have already submitted a [discussion topic](https://github.com/ropensci/crew.cluster/discussions) or [issue](https://github.com/ropensci/crew.cluster/issues) to discuss my idea with the maintainer.

# Related GitHub issues and pull requests

* Ref: #24 

# Summary

Make the slurm partition configurable for `crew_controller_slurm()`

### CODE REVIEW

**Comment by kkmann** [comment]
Creation of the extended class work fine, but the map() function complains about non-matching character lengths on an attribute check. Did not get to the bottom of it. 
image

File: R/crew_launcher_slurm.R [hunk 1]
```diff
@@ -49,6 +49,8 @@
 #'   SLURM cluster. `slurm_time_minutes = 60` translates to a line of
 #'   `#SBATCH --time=60` in the SLURM job script. `slurm_time_minutes = NULL`
 #'   omits this line.
+#' @param slurm_partition Character of length 1, name of the slurm partition to
+#'   create workers on.
```
Comment (wlandau) [inline R/crew_launcher_slurm.R:53]: Would be great to have a mention of how this translates into the `--partition` flag, as in the above `@param` descriptions. Also how a `NULL` value omits the flag.

**Comment by wlandau** [comment]
> Creation of the extended class work fine, but the map() function complains about non-matching character lengths on an attribute check. Did not get to the bottom of it.

You might need development `crew`.

**Comment by wlandau** [comment]
I am actually gearing up for a set of releases, so I will merge this now. Thanks!

**Pull request merged**